Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -563,232 +642,341 @@ def test_get_emails_for_activity___activity_leader(self):
-        self.url = hostURL + 'api/emails/activity/' + activity_code + '/group-admin/session/201809/'
+        self.session = \
+            self.createAuthorizedSession(leader_username, leader_password)
+        self.url = hostURL + 'api/emails/activity/' + activity_code + \
+            '/group-admin/session/201809/'
         response = api.get(self.session, self.url)
         if not response.status_code == 200:
-            pytest.fail('Expected 200 OK, got {0}.'.format(response.status_code))
+            pytest.fail('Expected 200 OK, got {0}.'\
+                .format(response.status_code))
         try:
             response.json()
         except ValueError:
-            pytest.fail('Expected Json response body, got {0}.'.format(response.text))
+            pytest.fail('Expected Json response body, got {0}.'\
+                .format(response.text))
         try:
             assert response.json()[0]['Email'] == "Chris.Carlson@gordon.edu"
         except KeyError:
-            pytest.fail('Expected Email in response, got{0}.'.format(response.json()))
+            pytest.fail('Expected Email in response, got{0}.'\
+                .format(response.json()))
 
-#    Verify that a supervisor can get the emails for any activity leader based on a session code
+#    Verify that a supervisor can get the emails for any activity leader based 
+#    on a session code
 #    Endpoint -- api/emails/activity/:id/leaders/session/:sessionCode
 #    Expected Status Code -- 200 OK
-#    Expected Respones Body -- Json response with a list of emails
+#    Expected Response Body -- Json response with a list of emails
     def test_get_emails_for_leaders___supervisor(self):
-        self.session = self.createAuthorizedSession(leader_username, leader_password)
-        self.url = hostURL + 'api/emails/activity/' + activity_code + '/leaders/session/201809/'
+        self.session = \
+            self.createAuthorizedSession(leader_username, leader_password)
+        self.url = hostURL + 'api/emails/activity/' + activity_code + \
+            '/leaders/session/201809/'
 
         response = api.get(self.session, self.url)
         if not response.status_code == 200:
-            pytest.fail('Expected 200 OK, got {0}.'.format(response.status_code))
+            pytest.fail('Expected 200 OK, got {0}.'\
+                .format(response.status_code))
         try:
             response.json()
         except ValueError:
-            pytest.fail('Expected Json response body, got {0}.'.format(response.text))
+            pytest.fail('Expected Json response body, got {0}.'\
+                .format(response.text))
         assert response.json()[0]['Email'] == "Emmy.Short@gordon.edu"
         assert response.json()[0]['FirstName'] == "Emmy"
         assert response.json()[0]['LastName'] == "Short"
 
-#    Verify that a leader can get the student's advisor based on sesson code.
+#    Verify that a leader can get the student's advisor based on session code.
```

Comment:
Should this say get the advisor for a student's involvement?  (I don't think it means academic advisor.)